Userland/Applications/Browser/History/HistoryModel.h

```diff
@@ -6,7 +6,7 @@
 
 #pragma once
 
-#include "Applications/Browser/History.h"
+#include "../History.h"
```

Comment:
:eyes: IMO relative include paths like this are worse than just writing `<Applications/Browser/History.h>`, but I don't really have a stronger argument than "yuck".

Reply:
Yeah, that particular line is awful. I'll go with the system-include syntax like you suggested.